src/index.js

```diff
@@ -161,15 +161,20 @@ module.exports = class Face3D {
         var isGLB = (resource.value.value || '').toLowerCase().indexOf(".v3d") == -1
 
         // Load scene
-        return Promise.resolve(this.vatomView.blockv.UserManager.encodeAssetProvider(resource.value.value || '')).then(resourceURL => 
-            isGLB 
-                ? this.loadGLTFScene(resourceURL) 
+        Promise.resolve(this.vatomView.blockv.UserManager.encodeAssetProvider(resource.value.value || '')).then(resourceURL =>
```

Comment:
```suggestion
        return Promise.resolve(this.vatomView.blockv.UserManager.encodeAssetProvider(resource.value.value || '')).then(resourceURL =>
```

Missing `return` here, if the model fails to load the error will not propagate back to the viewer